README.md

```diff
@@ -14,8 +10,279 @@ But we need have a clear way to discover how you've been invited to a group with
+
+
+```mermaid
+graph TB
+
+rootA(root)
+v1A(v1)
+4A(4) & dA(d)
+group-inviteA(group-invite):::groupInvite
+helsinkiA(helsinki):::group
+
+rootB(root)
+v1B(v1)
+9B(9) & cB(c)
+group-inviteB(group-invite):::groupInvite
+helsinkiB(helsinki):::group
+
+subgraph Staltz
+  rootA --> v1A --> 4A --> helsinkiA
+            v1A --> dA --> group-inviteA
+end
+
+subgraph Arj
+  rootB --> v1B --> 9B --> group-inviteB
+            v1B --> cB --> helsinkiB
+end
+
+classDef default stroke:none;
+classDef groupInvite fill: #BF2669, stroke:none, color:white;
+classDef group fill: #702A8C, stroke: none, color:white;
+```
+
+_Diagram showing updated state for Arj after he joins the group. Note the shards each feed lands in are random (but deterministic if you know the `groupKey`)_
+
+Staltz can see that Arj has accepted the invitation because he is able to decrypt the feed-announce message for Arj's "helsinki" feed, and read that the purpose is the `groupKey`.
+(He also knows where to watch for such a feed to be announced because the shared is deterministiclly chosen.)
+
+### 3. Non-group creator invites someone
+
+Arj now wants to invite Mix to the "helsinki" group. He follows the same pattern as in (2), but now as the inviter.
+
+Mix knows Arj is a part of the group because he was invited by them.
+Mix also knows staltz is part of the group because all `group/add-member` messages have
+```
+recps: [groupId, groupCreatorId, ...inviteeIds]
+```
+_As long as we know the creator we can always re-follow the chain of group-additions._
+
+:fire: TODO - we need to write the group spec up with these changes clearly somewhere.
+
+Staltz see Arj has invited Mix because he's replicating Arj's "group-invite" feed, so Statlz starts replicating Mix.
+
+
+
+## Questions
+
+1. if people in a group see that "mix was added" they are all going to start asking to replicate the feed "pathToMixsGroupFeed". On a network level this will cast a shadow revealing group members....
+    - possible soluton: raise sympathetic replication for obfuscation
+2. same problem as (1) exists when people join a group - they're going to start all asking for the same pattern of feeds (e.g. "can I have staltz/v1/d4/helsinki and arj/v1/c/helsinki"). It's like a fingerprint...
+
+---
 
 Existing work to be ported in:
 1. [2022-06-02 meeting](./2022-06-02-notes.md) - has good steps for how discovery works
 2. [2022-07-06 meeting](./2022-07-06-notes.md)
+
+
+
+
+<!-- References -->
+[ssb-meta-feeds-spec]: https://github.com/ssbc/ssb-meta-feeds-spec
+
+
+
+<!-- CSS -->
+<style>
+  .group-invite, .group {
+    display: inline-block;
+    width: 14px;
+    height: 14px;
+    border-radius: 2px;
+  }
+  .group-invite { background: #BF2669; }
+  .group { background: #702A8C; }
+
+  details > summary {
+    color: grey;
+    font-size: 12px
+  }
+  details > div {
+    font-style: italic;
+    background: #eee;
+
+    padding: 14px 10px;
+    border-left: 3px solid #999;
+    margin-left: 10px;
+  }
+</style>
```

Comment:
This is very pretty in the hackmd, but could we go for simpler markdown that works in GitHub (or GitLab, or wherever else we transport this markdown).

Reply:
yeah good idea. I didn't realise this would not work in GH. I added this style cos it looked butt ugly without. 
Figured if this didn't work it would fall back to the ugly?